src/FormFactory.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Opstalent\CrudBundle;
+
+use Opstalent\CrudBundle\Model\Field;
+use Opstalent\CrudBundle\Model\Form;
+use Opstalent\CrudBundle\Traits\MethodTrait;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * Class FormFactory
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class FormFactory
+{
+    use MethodTrait;
+
+    /**
+     * @var FormFactoryInterface
+     */
+    protected $formFactory;
+
+    /**
+     * @var FormBuilderInterface
+     */
+    protected $builder;
+
+    /**
+     * FormFactoryService constructor.
+     * @param FormFactoryInterface $formFactory
+     */
+    public function __construct(FormFactoryInterface $formFactory)
+    {
+        $this->formFactory = $formFactory;
+        $this->builder = $this->formFactory->createBuilder();
```

Comment:
do we need `$this->formFactory` in any other place? if not it has no sens to store it.